site/in-dev/gc.md

```diff
@@ -33,7 +33,7 @@ The Nessie GC tool can be downloaded from the [GitHub
 Releases](https://github.com/projectnessie/nessie/releases) page, for example:
 
 ```shell
-curl -L -o nessie-gc.jar https://github.com/projectnessie/nessie/releases/download/nessie-::NESSIE_VERSION::/nessie-gc-::NESSIE_VERSION::.jar
+curl -L -o nessie-gc.jar https://github.com/projectnessie/nessie/releases/download/nessie-::NESSIE_VERSION_BASE::/nessie-gc-::NESSIE_VERSION_BASE::.jar
```

Comment:
Obviously, this link will give 404. I don't think there is much we can do about it.

Reply:
I think it's better to just link the GC options on the downloads page and remove the curl example